src/getCoinCombination.test.js

```diff
@@ -7,4 +7,39 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it(`should return an array with zeros`, () => {
```

Comment:
Pay attention to the test names. We need to know exactly what we are testing. It is not quite clear what should be the input and the output... Should the function always return the array with zeros?